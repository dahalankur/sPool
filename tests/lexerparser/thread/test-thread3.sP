{ int x = f
int y = x
} 