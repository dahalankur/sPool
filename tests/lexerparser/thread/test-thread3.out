PROGRAM {[ 
    THREAD([ 
        DEFINE(false, INTTY, x, VAR(f)) 
        DEFINE(false, INTTY, y, VAR(x))])]}
